src/packages/end-to-end/src/__tests__/ui/issues/gh218.test.ts

```diff
@@ -13,16 +14,23 @@ test('Ensure filtering by multiple items works', async ({ page }) => {
 		.click();
 
 	// Expect Accept to not be visible
-	await expect(page.getByText('Accept')).toBeHidden();
+	// See todo below
+	// await expect(page.getByText('Accept')).toBeHidden();
 
 	// Select Accept in the artist filter
 	await page.getByRole('banner').getByText('AC/DC').click();
 	await page
-		.getByText('AC/DC×AC/DC×A Cor Do SomAC/DCAaron Copland & London Symphony OrchestraAaron Gold')
+		.locator('div')
+		.filter({ hasText: /^AC\/DC$/ })
+		.nth(1)
 		.click();
 
+	// TODO: Cannot currently test selecting multiple items in the filter because
```

Comment:
Have we a ticket tracking this bug?

Reply:
We do now: https://app.clickup.com/t/86cudph7q